src/config/dynamic-theme-fixes.config

```diff
@@ -7556,6 +7556,15 @@ CSS
 
 ================================
 
+deltahub.io
+
+CSS
+img {
```

Comment:
Why you add background to all images on the page when you can just INVERT the scrolling logos?

```
deltahub.io

INVERT
#ss-scrolling-logo-cloud a>img
```